lib/main.js

```diff
@@ -115,6 +115,8 @@ const Hydrogen = {
           this.handleKernelCommand({ command: "interrupt-kernel" }),
         "hydrogen:restart-kernel": () =>
           this.handleKernelCommand({ command: "restart-kernel" }),
+        "hydrogen:refresh-all-result-bubbles": () =>
```

Comment:
[INFO] What this command is doing is `cleanly-evaluate-code-from-top-to-bottom-and-update-existing-bubbles`, so we have to find better name of this verbose command name.